src/ports/postgres/modules/deep_learning/test/madlib_keras_evaluate.sql_in

```diff
@@ -59,3 +61,44 @@ ALTER TABLE keras_saved_out DROP COLUMN model_arch;
 SELECT assert(trap_error($TRAP$
 	SELECT madlib_keras_evaluate('keras_saved_out', 'cifar_10_sample_val', 'evaluate_out');
 	$TRAP$) = 1, 'Should error out if model_arch column is missing from model_table');
+
+
+
+m4_changequote(`<!', `!>')
+m4_ifdef(<!__POSTGRESQL__!>, <!!>, <!
+
+DROP TABLE IF EXISTS mst_table, mst_table_summary;
+SELECT load_model_selection_table(
+    'model_arch',
+    'mst_table',
+    ARRAY[1],
+    ARRAY[
+        $$loss='categorical_crossentropy',optimizer='Adam(lr=0.1)',metrics=['accuracy']$$,
+        $$loss='categorical_crossentropy', optimizer='Adam(lr=0.01)',metrics=['accuracy']$$,
+        $$loss='categorical_crossentropy',optimizer='Adam(lr=0.001)',metrics=['accuracy']$$
+    ],
+    ARRAY[
+        $$batch_size=5,epochs=1$$,
+        $$batch_size=10,epochs=1$$
+    ]
+);
+
```

Comment:
Do you think it makes sense to add test cases for the following scenarios ? (same goes for predict)

1. When mst_key is passed as NULL for multi model evaluate
1. When mst_key is passed as NON NULL for single model evaluate